tests/memory_checker/test_memory_checker.py

```diff
@@ -224,9 +224,14 @@ def remove_stress_utility(duthost, container_name):
     logger.info("BGP sessions are started up.")
 
 
+@pytest.fixture(autouse=True)
```

Comment:
how does this help? `enum_rand_one_per_hwsku_frontend_hostname` is module fixture buit rand_dut_for_all_functions is function level fixture, how does this work?